deploy/script/katello-debug.sh

```diff
@@ -61,6 +61,13 @@ fi
 add_files /etc/gofer
 add_files /var/log/gofer
 
+#foreman-tasks export
+if hash foreman-rake  2>/dev/null; then
+  echo "Exporting tasks, this may take a few minutes."
+  tasks_filename=`foreman-rake foreman_tasks:export_tasks | tail -n 1 | awk '{print $2}'`
```

Comment:
Is this not a default task? I tried testing this on my recent 2.2.1 and when I run this manually it tells me there is no task.